bounties/php/CMS/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1-Lack of rate limiting",
+    "DisclosureDate": "Aug 14",
+    "AffectedVersionRange": "< = 5.8.0",
+    "Summary": "no rate limit",
+    "Author": {
+        "Username": "B3EF",
+        "Name": "Ajmal Aboobacker"
+    },
+    "Package": {
+        "Registry": "php",
+        "Name": "Lavalite/cms",
+        "URL": " https://lavalite.org/"
+    },
+    "CWEs": [{
+        "ID": "",
```

Comment:
As it's the **Lack of Rate Limiting** on a login page, set it to **CWE-307**.

:books: **Reference:** CWE-307